demo/test.c

```diff
@@ -2325,6 +2325,140 @@ static int test_mp_radix_size(void)
    return EXIT_FAILURE;
 }
 
+#ifndef S_MP_DIV_SMALL
```

Comment:
Remove the guard and test both functions test_s_mp_div_recursive, trst_s_mp_div_small. Additional guards are rarely necessary. MP_HAS takes care of it.